.github/ISSUE_TEMPLATE/02-documentation.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "Handsontable"
+about: Issues regarding Handsontable documentation.
+title: ""
+labels: "Documentation"
```

Comment:
There is no such label. For wrappers there is `"angular"`, `"react"`, or `"vue"`

![image](https://user-images.githubusercontent.com/571316/124888628-8efef100-dfd6-11eb-8c8b-cf068a60aa26.png)

I propose to change that to `labels: "Type: Documentation"`

![image](https://user-images.githubusercontent.com/571316/124888787-b786eb00-dfd6-11eb-8c9c-8c8e34aab0bd.png)

Or to be consistent with wrappers change the label name to `"documentation"` and then change the already created label. 

@wojciechczerniak?

Reply:
I agree, Documentation will be now part of monorepo and can look like other projects. +1 on renaming the label.

Although, @AMBudnik keeps tabs on labels. I'll ask

Reply:
👍 for the new name

Reply:
Done in 208dc9bd734dae85aab32e1cc8d432cae792b2d2.